scripts/migration/remove_invalid_social_entries.py

```diff
@@ -0,0 +1,43 @@
+"""
```

Comment:
Ugh, sorry. I missed that, thought I had read move to scripts/migration. Ok, back to the drawing board.